src/components/Nav/Dropdown/Dropdown.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import DropdownContent from './DropdownContent/DropdownContent';
+import './Dropdown.scss';
+
+function Dropdown() {
+  const [menuMouseEnter, setMenuMouseEnter] = useState('Categories');
+
+  return (
+    <div className="Dropdown">
```

Comment:
- 전체적으로 최상위 태그의 className을 컴포넌트 이름과 동일하게 부여한 뒤 그 className하위로 네스팅 하는 부분이 적용안되어있습니다.
- CRA로는 index.html이 하나인 SPA프로젝트를 구성하는 것이고 이런 상황에서 결과적으로 모든 css파일들은 하나의 html에 한번에 불러와집니다.
- 결과적으로 모든 css파일이 한번에 불러와지기에 이 css파일에서 부여한 스타일이 동일한 className을 가진 다른 컴포넌트의 요소들에게도 영향을 미칠 수 있습니다.
- 따라서 반드시 **최상위 태그의 className을 컴포넌트 이름과 동일하게 부여한 뒤 그 className하위로 네스팅** 적용해주세요
- 해당 내용에 맞게 수정하시면, scss 파일 부분도 수정해주세요!
```suggestion
    <div className="dropdown">
```

Reply:
리뷰 반영했습니다!